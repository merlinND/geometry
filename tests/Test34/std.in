C myCircle 0 0 1
C mySecondCircle 1 1 42
C myThirdCircle 0 0 44
C myFourthCircle 0 0 21
C myFifthCircle 1 1 489
C mySeventhCircle 0 0 99
C mySixthCircle 1 1 27
C myEightCircle 0 0 1000
C myNinthCircle 0 1 6
C myTenthCircle 1 1 2
R myRectangle 0 0 1 1
R mySecondRectangle 1 1 4 2
R myThirdRectangle 0 0 4 4
R myFourthRectangle 0 0 2 1
R myFifthRectangle 1 1 48 9
R mySixthRectangle 1 1 2 7
R mySeventhRectangle 0 0 9 9
R myEightRectangle 0 0 10 0
R myNinthRectangle 0 1 6 9
R myTenthRectangle 1 1 42 12
LIST
UNDO
UNDO
REDO
REDO
LIST
EXIT
