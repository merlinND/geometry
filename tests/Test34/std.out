OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
C myCircle 0 0 1
C myEightCircle 0 0 1000
R myEightRectangle 0 0 10 0
C myFifthCircle 1 1 489
R myFifthRectangle 1 1 48 9
C myFourthCircle 0 0 21
R myFourthRectangle 0 0 2 1
C myNinthCircle 0 1 6
R myNinthRectangle 0 1 6 9
R myRectangle 0 0 1 1
C mySecondCircle 1 1 42
R mySecondRectangle 1 1 4 2
C mySeventhCircle 0 0 99
R mySeventhRectangle 0 0 9 9
C mySixthCircle 1 1 27
R mySixthRectangle 1 1 2 7
C myTenthCircle 1 1 2
R myTenthRectangle 1 1 42 12
C myThirdCircle 0 0 44
R myThirdRectangle 0 0 4 4
OK
OK
OK
OK
C myCircle 0 0 1
C myEightCircle 0 0 1000
R myEightRectangle 0 0 10 0
C myFifthCircle 1 1 489
R myFifthRectangle 1 1 48 9
C myFourthCircle 0 0 21
R myFourthRectangle 0 0 2 1
C myNinthCircle 0 1 6
R myNinthRectangle 0 1 6 9
R myRectangle 0 0 1 1
C mySecondCircle 1 1 42
R mySecondRectangle 1 1 4 2
C mySeventhCircle 0 0 99
R mySeventhRectangle 0 0 9 9
C mySixthCircle 1 1 27
R mySixthRectangle 1 1 2 7
C myTenthCircle 1 1 2
R myTenthRectangle 1 1 42 12
C myThirdCircle 0 0 44
R myThirdRectangle 0 0 4 4
